mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -937,11 +940,12 @@ def Vector_OuterProductOp :
     lowered to the LLVMIR dialect, this form emits `llvm.intr.fma`, which
     is guaranteed to lower to actual `fma` instructions on x86.
 
-    An optional kind attribute may be specified to be add/mul/min/max
-    for int/fp, and and/or/xor for int only. The default is "add", in which
-    case the operation returns a fused multiply-add. In other cases it returns
-    a multiply followed by the appropriate operation (for example, a compare and
-    select for "max").
+    An optional kind attribute may be specified to be: `add`/`mul`/`minsi`
+    /`minui`/`maxsi`/`maxui`/`and`/`or`/`xor` for integers, and `add`/`mul`
+    /`minf`/`maxf`/`minimumf`/`maximumf` for floats.
+    The default is `add`, in which case the operation returns a fused
+    multiply-add. In other cases it returns a multiply followed by the
+    appropriate operation (for example, a compare and select for `maxf`).
```

Comment:
I think the "compare and select" is not longer accurate?

Reply:
what should it be instead?  I'm not familiar with this one

Reply:
This can be a follow up fix -- just let me know what wording you'd like to see here

Reply:
I would remove everything after "The default is `add`. "The operation returns a fused multiply-add" seems to be talking about the lowering. Same for the `maxf`. We generate one of the max operations in the arith dialect. THis is probably something that was written quite some time ago.

Reply:
https://github.com/llvm/llvm-project/pull/70824